drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -1999,10 +2032,25 @@ void deserialize(BufReader &p_reader) {
 #endif
 
 #if TARGET_OS_IOS
-	msl_options.ios_use_simdgroup_functions = (*metal_device_properties).features.simdPermute;
+	msl_options.ios_use_simdgroup_functions = (*device_properties).features.simdPermute;
 #endif
 
-	msl_options.argument_buffers = true;
+	bool disable_argument_buffers = false;
+	if (String v = OS::get_singleton()->get_environment(U"GODOT_DISABLE_ARGUMENT_BUFFERS"); v == U"1") {
+		disable_argument_buffers = true;
+	}
```

Comment:
It's undocumented, and I use this for debugging purposes. I was able to test out argument buffers for a number of projects and they ran without issue